rocthrust/PKGBUILD

```diff
@@ -1,50 +1,36 @@
 # Maintainer: Markus Näther <naetherm@informatik.uni-freiburg.de>
 pkgname=rocthrust
 pkgver=3.3.0
-pkgrel=3
-pkgdesc="Port of the Thrust parallel algorithm library atop HIP/ROCm."
+pkgrel=4
+pkgdesc="Port of the Thrust parallel algorithm library atop HIP/ROCm"
 arch=('x86_64')
 url="https://github.com/ROCmSoftwarePlatform/rocThrust"
-license=('custom:NCSAOSL')
-depends=('hcc' 'hip')
-makedepends=('cmake' 'hcc' 'python2' 'rocminfo' 'comgr')
-source=("https://github.com/ROCmSoftwarePlatform/rocThrust/archive/rocm-$pkgver.tar.gz")
+license=('Apache-2.0')
+depends=('hip-hcc' 'rocprim')
+makedepends=('cmake' 'git' 'hcc')
+source=("$url/archive/rocm-$pkgver.tar.gz")
```

Comment:
For the source can we add `${pkgname}-${pkgver}`? Like in the warning box here: https://wiki.archlinux.org/index.php/PKGBUILD#source

Reply:
Good point. We should change this for all packages.